src/components/CalendarEvent.js

```diff
@@ -25,7 +25,12 @@ class CalendarEvent extends React.Component {
 
     return (
             <div className={eventClasses}
-                 onClick={this.props.onClick.bind(...sharedArguments)}
+              onClick={
+                (e) => {
+                  this.props.onClick.bind(...sharedArguments)();
```

Comment:
No need to use `bind` here since you are already creating a new function body. This is just unnecessary overhead.

Instead, you could do something like the following:

```js
onClick={e => {
  const [ , ...args ] = sharedArguments;
  this.props.onClick(...args);
  e.stopPropagation();
}}
```